WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -1179,37 +1232,93 @@ static GSourceFuncs _handlerIntervention =
 
             _adminLock.Unlock();
         }
+        void Register(Exchange::IBrowser::INotification* sink) override
```

Comment:
Isn't IWebBrowser::INotification enough? do we need both IWebBrowser and IBrowser?

Reply:
Yes by definition. We do live by the rule: an interface can be extended, not changed. Backwards compatibility is crucial. Our DIALServer, for example is still using IBrowser/IBrowser::INotification. We can not break that!

Reply:
Other example, a customer is using the ESPIAL Browser (which we wrapped in a plugin, like we do WebKit, we also have somewhere the QTBrowser wrapped with this same interface, but that is not used anywhere) this is also exposing the IBrowser/INotification interfaces. The customer build its own logic (statecontroller) on top of these browser interface (he is running the ESPIAL and the WebKitBrowser) So this interface is published and deployed. Breaking it is not good!